regolith/helpers/l_generalhelper.py

```diff
@@ -0,0 +1,84 @@
+"""Helper for finding people's information in the people collection
+"""
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    search_collection,
+    collection_str,
+)
+HELPER_TARGET = "lister"
+
+def subparser(subpi):
+    subpi.add_argument(
+        "coll",
+        help='collection where the lister is going to run.')
+    subpi.add_argument(
+        "--kv_filter",
```

Comment:
this needs a short name, like -f (for filter) (long name is ok as is)

Reply:
done!